src/zenml/utils/pipeline_docker_image_builder.py

```diff
@@ -381,20 +381,20 @@ def build_and_push_docker_image(
             # Leave the build context empty if we don't want to copy any files
             requires_build_context = (
                 docker_configuration.copy_files
-                or docker_configuration.copy_profile
+                or docker_configuration.copy_global_config
             )
             build_context_root = (
                 source_utils.get_source_root_path()
                 if requires_build_context
                 else None
             )
-            maybe_include_active_profile = (
-                _include_active_profile(build_context_root=build_context_root)  # type: ignore[arg-type]
-                if docker_configuration.copy_profile
+            maybe_include_global_config = (
```

Comment:
what is `maybe_include_global_config`? :D

Reply:
this is an option part of the docker image build configuration that copies the global configuration, including the local zen store contents, into the docker image. It replaces the previous option that was used to copy the active profile along with the active stack and its components into the docker image.